comfy_cli/command/workflow_edit.py

```
@@ -0,0 +1,537 @@
+    class_type: Annotated[str, typer.Argument(help="Node class_type, e.g. KSampler.")],
+    at: Annotated[
+        str | None,
+        typer.Option("--at", show_default=False, help="Canvas position 'x,y' for the new node."),
+    ] = None,
+    actor: Annotated[str, typer.Option("--actor", help="Op author id (for CRDT stamping).")] = "cli",
+    base_version: Annotated[int, typer.Option("--base-version", help="Draft version this edit is based on.")] = 0,
+    stdout: Annotated[bool, typer.Option("--stdout/--in-place", show_default=False)] = False,
+    input_path: Annotated[str | None, typer.Option("--input", show_default=False)] = None,
+    host: Annotated[str | None, typer.Option(show_default=False)] = None,
+    port: Annotated[int | None, typer.Option(show_default=False)] = None,
+    where: Annotated[str | None, typer.Option("--where", show_default=False, help="Catalog target: local | cloud.")] = None,
```

Pylint (4.0.6)

[convention] Line too long (124/120)

(C0301)